theano/compile/ops.py

```diff
@@ -518,6 +518,8 @@ class FromFunctionOp(gof.Op):
 
     """
 
+    __props__ = ("fn", "itypes", "otypes", "infer_shape")
```

Comment:
I'm not sure this is good. infer_shape is the parameter in the \__init__, but usually, we set the props with the same name in the class. We don't do that here.

For simplicity, just remove.